.github/workflows/build_and_publish.yml

```diff
@@ -55,3 +55,17 @@ jobs:
         password: ${{ secrets.DOCKER_PASSWORD }}
         tags: "latest,${{ github.event.release.tag_name }}"
 
+ deploy-docs:
+   name: Deploy Docs to GitHubIO
+   runs-on: ubuntu-latest
+   steps:
+    - name: Checkout main
+      uses: actions/checkout@v2
+
+    - name: Deploy docs
+      uses: mhausenblas/mkdocs-deploy-gh-pages@master
```

Comment:
The sh file runned by this action looks quite complicated. I wonder if it wouldn't be easier to just run the usual commands as "run" in here instead. But fine if it works!